src/graphics/reproject-texture.js

```diff
@@ -98,11 +110,12 @@ function reprojectTexture(source, target, options = {}) {
         "#define SOURCE_FUNC " + sourceFunc + "\n" +
         "#define TARGET_FUNC " + targetFunc + "\n" +
         "#define NUM_SAMPLES " + numSamples + "\n" +
-        "#define NUM_SAMPLES_SQRT " + Math.round(Math.sqrt(numSamples)).toFixed(1) + "\n\n" +
+        "#define NUM_SAMPLES_SQRT " + Math.round(Math.sqrt(numSamples)).toFixed(1) + "\n" +
+        "#define SUPPORTS_TEXLOD " + (device.extTextureLod ? '1' : '0') + '\n\n' +
         shaderChunks.reprojectPS,
         processFunc + decodeFunc + encodeFunc + sourceFunc + targetFunc,
         null,
-        device.webgl2 ? "" : "#extension GL_OES_standard_derivatives: enable\n"
+        device.webgl2 ? "" : "#extension GL_OES_standard_derivatives: enable\n" + (device.extTextureLod ? "#extension GL_EXT_shader_texture_lod: enable\n" : "")
```

Comment:
and this should probably be moved to some function on a device .. that would return the string to append, as it's likely used in other places?

Reply:
yeah agree, though probably not something I'd want to do in this PR.